v2/internal/frontend/runtime/dev/main.js

```diff
@@ -80,7 +80,7 @@ function handleDisconnect() {
 
 function _connect() {
     if (websocket == null) {
-        websocket = new WebSocket('ws://' + window.location.host + '/wails/ipc');
+        websocket = new WebSocket((window.location.protocol.startsWith("https") ? "wss://" : "ws://") + window.location.host + "/wails/ipc");
         websocket.onopen = handleConnect;
```

Comment:
Maybe we should move this logic into a function that simply returns the connection string.  Would clean up this particular location in the code.   Presumably it only impacts 'dev' and 'server' type modes.

Reply:
Could drop the ternary operator to keep the line short:

```
let proto = 'ws:';
if (window.location.protocol == 'https:') {
    proto = 'wss:';
}
websocket = new WebSocket(proto + '//' + window.location.host + '/wails/ipc');
```